test/ci/amd64.sh

```diff
@@ -5,10 +5,17 @@ export SSSE3_CFLAGS=-mssse3
 export SSE41_CFLAGS=-msse4.1
 export SSE42_CFLAGS=-msse4.2
 export AVX_CFLAGS=-mavx
-export AVX2_CFLAGS=-mavx2
+# no AVX2 on GHA macOS
+if [ "${OBJCOPY:-}" != "echo" ]; then
```

Comment:
This is a bit of a hack. It's probably nicer to do OS detection than to rely on `OBJCOPY` being set to the magic string `"echo"`.

Reply:
done